src/Common/AbstractModel.m

```diff
@@ -139,9 +139,12 @@ function saveObj(obj, suffix)
             objectProperties = fieldnames(objStruct);
             %Loop through all object properties
             for propIndex = 1:length(objectProperties)
-
+                  
                 % Assign object property value to identically named struct field.
+                if ismember(objectProperties{propIndex},fieldnames(loadedStruct))
```

Comment:
Please offer another solution if this is against a decision you made. The cases where a fieldname was changed or an attribute was removed were OK, but had to do this to add a new attribute.

Reply:
This question to us isn't clear to me. What's the potential conflict here that you're asking us to check?

Reply:
Before adding this ‘ismember’ logic, it was erroring given that the new object property added was not present in the loaded model. As the comment above said identically named struct, I think this was a sane check. I just wanted to make sure that I did not bypass anything that was intended to throw an exception.

Reply:
I still don't fully understand what the risk of this change might be. If this branch passes my PR checklist & travis tests, I don't see what problem it might cause. Unless you were wondering if you should write is as a patch or not?